dist/legacy-assets/js/main.js

```diff
@@ -295,6 +308,9 @@ function getPageData(collectionId, path, success, error) {
     dataType: 'json',
     type: 'GET',
     success: function (response) {
+      var date = new Date();
+      date = date.getHours() + ":" + date.getMinutes() + "." + date.getMilliseconds();
```

Comment:
this line is repeated quite a lot - could be worth creating a log function which does it for you?